packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -19,16 +21,22 @@ import ChatHeader from './header';
 import type {
   MessageSendEvent as MessageBoxMessageSendEvent,
   Properties as MessageBoxProperties,
+  TypingStartEvent as MessageBoxTypingStartEvent,
 } from './messagebox';
 import MessageBox from './messagebox';
 import MessageList from './messagelist';
 
 const CHAT_CLASS = 'dx-chat';
 const TEXTEDITOR_INPUT_CLASS = 'dx-texteditor-input';
 
+type TypingStartEvent = NativeEventInfo<Chat> & { user?: User };
+type TypingEndEvent = NativeEventInfo<Chat> & { user?: User };
```

Comment:
Why types for start and end are the same although there it no `event` parameter in `end` handler?

Reply:
I already improve typing in the next PR